cmd/plural/vpn.go

```diff
@@ -0,0 +1,290 @@
+				},
+			},
+		},
+		{
+			Name:      "delete-client",
+			ArgsUsage: "NAME",
+			Usage:     "create a new vpn client for a server",
+			Action:    requireArgs(p.handleWireguardPeerDelete, []string{"NAME"}),
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "server",
+					Usage: "the vpn server to delete the clients from",
+				},
+			},
+		},
+		{
+			Name:      "client-config",
+			ArgsUsage: "NAME",
+			Usage:     "get the config for a vpn client for a server",
+			Action:    requireArgs(p.handleWireguardPeerConfig, []string{"NAME"}),
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "server",
+					Usage: "the vpn server to get the client config from",
+				},
+				cli.StringFlag{
+					Name:  "path",
+					Usage: "output path for the config wireguard client config. the filename will be NAME.conf",
+				},
+			},
+		},
+	}
+}
+
+func (p *Plural) handleWireguardServerList(c *cli.Context) error {
+	conf := config.Read()
+	if err := p.InitKube(); err != nil {
+		return utils.HighlightError(err)
+	}
+
+	if err := p.checkIfVPNInstalled(); err != nil {
+		return utils.HighlightError(fmt.Errorf("wireguard is not installed. run `plural bundle list wireguard` to find the bundle to install"))
+	}
+
+	servers, err := vpn.ListServers(p.Kube, conf.Namespace(wireguardNamespace))
+	if err != nil {
+		return utils.HighlightError(err)
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Hostname", "Port", "Ready"})
+	for _, s := range servers.Items {
+		table.Append([]string{s.Name, s.Status.Hostname, s.Status.Port, strconv.FormatBool(s.Status.Ready)})
+	}
+	table.Render()
+	return nil
+}
+
+func (p *Plural) handleWireguardPeerList(c *cli.Context) error {
+	var server string
+	server = wireguardServerName
+	if c.String("server") != "" {
+		server = c.String("server")
+	}
+
+	conf := config.Read()
+	if err := p.InitKube(); err != nil {
+		return utils.HighlightError(err)
+	}
+
+	if err := p.checkIfVPNInstalled(); err != nil {
+		return utils.HighlightError(fmt.Errorf("wireguard is not installed. run `plural bundle list wireguard` to find the bundle to install"))
+	}
+
+	peers, err := vpn.ListPeers(p.Kube, conf.Namespace(wireguardNamespace))
+	if err != nil {
+		return utils.HighlightError(err)
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Address", "Config Secret", "Public Key", "Ready"})
+	for _, p := range peers.Items {
+		if p.Spec.WireguardRef == server {
+			table.Append([]string{
+				p.Name,
+				p.Spec.Address,
+				p.Status.ConfigRef.Name,
+				p.Spec.PublicKey,
+				strconv.FormatBool(p.Status.Ready),
+			})
+		}
+	}
+	table.Render()
+	return nil
+}
+
+func (p *Plural) handleWireguardPeerCreate(c *cli.Context) error {
+	var serverName string
+	serverName = wireguardServerName
+	if c.String("server") != "" {
+		serverName = c.String("server")
+	}
+
+	name := c.Args().Get(0)
+	conf := config.Read()
+	if err := p.InitKube(); err != nil {
```

Comment:
A lot of these functions seem to init kube and checkIfVPNInstalled, perhaps we should make a wrapper func like we have for some other stuff like `rooted` elsewhere to remove duplicate code.